src/main/java/hudson/plugins/ec2/EC2Cloud.java

```diff
@@ -1135,16 +1160,22 @@ public FormValidation doCheckSshKeysCredentialsId(@AncestorInPath ItemGroup cont
                 // Don't do anything if the user is only reading the configuration
                 return FormValidation.ok();
             }
-            if (value == null || value.isEmpty()){
-                return FormValidation.error("No ssh credentials selected");
-            }
 
-            SSHUserPrivateKey sshCredential = getSshCredential(value, context);
             String privateKey = "";
```

Comment:
```suggestion
            String privateKey;
```
?

Reply:
that was my bad, i just shifted the original code around and didn't catch that